README.md

```diff
@@ -223,109 +223,37 @@ index.search(
 
-#### Creating a Custom `GsonJsonHandler` <!-- omit in toc -->
+*Notes*: We strongly recommend using the `Gson` library.
 
-To create a custom JSON handler, create an object of GsonJsonHandler and send the GSON object in the parameterized constructor.<br>
+#### Use a Custom `JacksonJsonHandler` <!-- omit in toc -->
 
-```java
-Gson gson = new GsonBuilder()
-             .disableHtmlEscaping()
-             .setFieldNamingPolicy(FieldNamingPolicy.UPPER_CAMEL_CASE)
-             .setPrettyPrinting()
-             .serializeNulls()
-             .create();
-private GsonJsonHandler jsonGson = new GsonJsonHandler(gson);
-jsonGson.encode("your_data");
-```
-
-#### Creating a Custom `JacksonJsonHandler` <!-- omit in toc -->
-
-Another method is to create an object of `JacksonJsonHandler` and set the required parameters. The supported option is an object of `ObjectMapper`. It's passed as a parameter to the `JacksonJsonHandler`’s parameterized constructor. This is used to initialize the mapper variable.
-
-The mapper variable is responsible for the encoding and decoding of the JSON.
-
-Using the custom JSON:
+Using the `jackson json` Library:
+Just create a `JacksonJsonHandler` object and send it through the `Config` object to the parameterized client.
 
 ```java
-Config config = new Config("http://localhost:7700", "masterKey");
-HttpAsyncClient client = HttpAsyncClients.createDefault();
-ApacheHttpClient client = new ApacheHttpClient(config, client);
-private final JsonHandler jsonHandler = new JacksonJsonHandler(new ObjectMapper());
-private final RequestFactory requestFactory = new BasicRequestFactory(jsonHandler);
-private final GenericServiceTemplate serviceTemplate = new GenericServiceTemplate(client, jsonHandler, requestFactory);
-
-private final ServiceTemplate serviceTemplate;
-serviceTemplate.getProcessor().encode("your_data");
-```
+import com.meilisearch.sdk.json.JacksonJsonHandler;
 
-### Creating a Custom `JsonbJsonHandler <!-- omit in toc -->
-
-Another method of creating a JSON handler is to create an object of `JsonbJsonHandler` and send the `Jsonb` object to the parameterized constructor.
-
-```java
-Jsonb jsonb = JsonbBuilder.create();
-private JsonbJsonHandler jsonbHandler = new JsonbJsonHandler(jsonb);
-jsonbHandler.encode("your_data");
+Config config = new Config("http://localhost:7700", "masterKey", new JacksonJsonHandler());
+Client client = new Client(config);
 ```
 
-### Custom Client <!-- omit in toc -->
+#### Use a Custom `JsonHandler` <!-- omit in toc -->
 
-To create a custom `Client` handler, create an object of `Client` and set the required parameters.
-
-A `Config` object should be passed, containing your host URL and your API key.
+To create your own `JsonHandler`, you will need to make it implement the two methods of the `JsonHandler` class. Then pass it to the `Client`.
```

Comment:
```suggestion
To create your own `JsonHandler`, it has to implement the two methods of the `JsonHandler` class. Then pass it to the `Client`.
```

> implement the two methods of the `JsonHandler` class

Is this not based on an interface? If it is we should ask the user to comply to the interface

Reply:
Yes it is! I will change that